### ISSUE TITLE
Feature request – DNS intercept for ctrld

### ISSUE THREAD

**Issue by pmcarrion** [Control-D-Inc/ctrld#56]
Some devices and apps are hard-coded to use a certain DNS server (usually Google DNS).
Please add a **DNS intercept** feature to ctrld.

**Comment by yegors** [comment]
This is already on the roadmap for next release (v1.2.2) after v1.2.1

**Comment by pmcarrion** [comment]
That's great news!
Will it work on computers and IPv6 networks as well?

**Comment by yegors** [comment]
I don't see why not.

**Comment by paulg1981** [comment]
Has this been implemented in the latest version 1.3.3?

**Comment by cuonglm** [comment]
> Has this been implemented in the latest version 1.3.3? 

The latest version is v1.3.4, FYI.

This feature has not been implemented yet.

**Comment by gniting** [comment]
Adding a +1 for this. At present I achieve this via firewall rules. Would be nice to centralize all DNS matters to live within the ctrld setup.

**Comment by jinkang23** [comment]
Would be awesome if this support is added to contrld proxy tool. 

Any update on when it will be added?

**Comment by mvevitsis** [comment]
+1

**Comment by gniting** [comment]
@yegors any guidance on timeline?

**Comment by yegors** [comment]
Been pushing it off to work on quality of life things and performance improvements. It's on the board for v1.3.11, but I can't guarantee that it will be released in the next update.

**Comment by gniting** [comment]
Yep, saw that. Nice work. Waiting to see it be released for linux/pfSense/etc next.
